### PR TITLE
SCI-3649 All reagent data from step are not imported

### PR DESCRIPTION
**Pull request by radiokills**
Jira ticket: [SCI-3649](https://biosistemika.atlassian.net/browse/SCI-3649)

### What was done
Adding catalog number, fixing up the markup. (Urban will be buying a beer until the end of the summer for what he did :) )

### CODE REVIEW

File: app/views/protocol_importers/templates/_details.html.erb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <p>
-  <%= item[:details]&.any? ? "<b>#{t('protocol_importers.templates.details.title') }</b>" : '' %>
-  <% item[:details]&.each do |k, v| %>
-    <%= "#{k.camelize.to_s}: #{v.to_s}" %>
+  <% if item[:details]&.any? %>
+    <b> <%= t('protocol_importers.templates.details.title')%>: </b>
+    <br />
+    <% item[:details]&.each do |k, v| %>
+      <%= "#{k.humanize}: #{v.to_s}" %>
```
[the remark this answers is not in the document]
Comment (radiokills) [reply app/views/protocol_importers/templates/_details.html.erb]: `.compact` for the save!

[the remark this answers is not in the document]
Comment (jbargu) [reply app/views/protocol_importers/templates/_details.html.erb]: image
It still looks like this. Not sure if this is desirable. Also note, Molecular Weight in Reagent, it should be removed.

[the remark this answers is not in the document]
Comment (radiokills) [reply app/views/protocol_importers/templates/_details.html.erb]: damn!

File: app/views/protocol_importers/templates/_details.html.erb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <p>
-  <%= item[:details]&.any? ? "<b>#{t('protocol_importers.templates.details.title') }</b>" : '' %>
-  <% item[:details]&.each do |k, v| %>
-    <%= "#{k.camelize.to_s}: #{v.to_s}" %>
+  <% if item[:details]&.any? %>
+    <b> <%= t('protocol_importers.templates.details.title')%>: </b>
+    <br />
+    <% item[:details].reject { |_,v| v.blank? }.each do |k, v| %>
```
Comment (jbargu) [inline app/views/protocol_importers/templates/_details.html.erb]: This should most likely check for `zero?` as well

**Pull request merged**